README.md

```diff
@@ -85,6 +85,8 @@ Equivalent Caddyfile (there is not currently a way to customize IP sources via C
 			example.net subdomain
 		}
 		check_interval 5m
+		ip_source upnp
+		ip_source endpoint https://icanhazip.com
```

Comment:
I think it would be helpful to show that multiple IP source endpoints can be configured. Can you add another here?